packages/core/useBreakpoints/index.ts

```diff
@@ -46,27 +48,35 @@ export function useBreakpoints<K extends string>(
 
   const { window = defaultWindow, strategy = 'min-width', ssrWidth = useSSRWidth() } = options
 
-  const ssrSupport = typeof ssrWidth === 'number'
-  const mounted = ssrSupport ? ref(false) : { value: true }
-  if (ssrSupport) {
-    tryOnMounted(() => mounted.value = !!window)
+  const ssrSupport = ref(getCurrentInstance()?.root.isMounted ? typeof ssrWidth === 'number' : false)
```

Comment:
Probably I don't fully understand the intention but shouldn't this be inverted?

```
const ssrSupport = ref(getCurrentInstance()?.root.isMounted ? false : typeof ssrWidth === 'number')
```

`getCurrentInstance()?.root.isMounted` means that hydration is finished, right? In this case `ssrSupport` should be disabled, shoudn't it?

Reply:
Yep